codecov.yml

```diff
@@ -0,0 +1,3 @@
+codecov:
+  notify:
+    require_ci_to_pass: no
```

Comment:
Do you know what would happen if there was a major failure, and say, just the very first test was executed?  Would that bring the coverage numbers down?

Reply:
@ldoktor had a question here for you.

Reply:
IIUC it has nothing to do with the coverage run, but with the rest of github checks. I noticed the Codecov report only after all checks are successfully performed.

Reply:
... as for the coverage, it's up to us. We call the "upload" results so we can decide whether to do it on incomplete runs or not.